docs/UserGuide.md

```diff
@@ -3,271 +3,560 @@ layout: page
 
+[Return to Table of Contents](#table-of-contents)
+<br><br>
 
-### Adding a person: `add`
 
-Adds a client to ClientBook.
+### `add`: Add client contact
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [i/POLICY_ID[>POLICY_URL]] [t/TAG]…​`
+**Purpose**: Adds a client as contact to ClientBook.
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags and insurance policies (including 0)
+**Format**: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [i/POLICY_ID[>POLICY_URL]] [t/TAG]…​`
+
+<div markdown="block" class="alert alert-info">
+:bulb: 
+**Tip**:
+A person can have any number of tags and insurance policies (including 0).
 </div>
 
-* It is perfectly fine to not include the URL to the insurance policy document!
+* It is optional to include the `POLICY_URL` for the specified `POLICY_ID`.
 * To include the URL, remember to use '>' to indicate that a particular insurance policy is linked to a URL, as shown in the second example below.
 
-Examples:
-* (example of a client with insurance policy but no URL)`add n/John Doe p/98765432 e/johnd@example.com a/Orchard i/Policy_1273 t/basic`
-* (example of a client with insurance policy and URL)`add n/Tom Tan p/91234567 e/tomt@example.com a/Orchard i/Policy_1274>www.myinsurancecompany.com/policy_1274 t/basic`
-* (example with no insurance policy and no tag)`add n/Betsy Crowe e/betsycrowe@example.com a/Kent Ridge`
+**Examples**:
+* Example of a client with insurance policy but no URL associated with insurance policy
+    * `add n/John Doe p/98765432 e/johnd@email.com a/Orchard i/Pol_#123456 t/basic`
+* Example of a client with insurance policy and URL associated with insurance policy
+    * `add n/Tom Tan p/91234567 e/tomt@email.com a/Orchard i/Policy_1274>www.myinsurancecompany.com/policy_1274 t/basic`
+* Example with no insurance policy and no tag
+    * `add n/Betsy Crowe e/betsycrowe@email.com a/Kent Ridge`
 
-### Listing all persons : `list [-ATTRIBUTE]`
+[Return to Table of Contents](#table-of-contents)
+<br><br>
 
-![list_with_filter](images/list_with_filter.png)
 
-Shows a list of all clients in ClientBook. An optional attribute option can be added to show a list with only the specified attribute.
+### `edit`: Edit client contact 
 
-Format: `list [-ATTRIBUTE]…​`
+**Purpose**: Edits an existing client contact in the ClientBook.
 
-Examples:
-* `list` Shows a list of all clients and all their information
-* `list -policy` Shows a list of all clients and their policy numbers
-* `list -phone -policy` Shows a list of all clients, their phone number, and their policy numbers
+**Format**: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/POLICY_NUMBER] [t/TAG]…​`
 
-### Editing a person : `edit`
+* Edits the client at the specified `INDEX`.
+    * `INDEX` refers to the index number shown in the displayed client list.
+    * `INDEX` must be more than 1, and less than or equal to the index of the last item in the displayed list.
```

Comment:
This line is a bit confusing, may be instead of "more than 1", it is more suitable to be "at least 1"? Alternatively, we can also say "index must be within the numbers listed on the clientbook"?